addons/sale/tests/test_performance.py

```diff
@@ -1,15 +0,0 @@
-from odoo.tests import tagged
-
-from odoo.addons.base.tests.common import PerformanceCommon
-
-
-@tagged('post_install', '-at_install')
-class TestSalePerformance(PerformanceCommon):
```

Comment:
Why this diff ?

Reply:
The test will be applied to all the models, so I removed the test redundancy.